### ISSUE TITLE
Readme could use a little revamp

### ISSUE THREAD

**Issue by bogdan-calapod** [hufman/AndroidAutoIdrive#134]
I think the Readme could use a bit of a revamp - the supported app list for example could be presented in a more pleasant way, such as a table or something ?

I have a PR in progress re: this, will push it tomorrow 😁

**Comment by hufman** [comment]
Good idea! Perhaps some of the more detailed information could be moved to the docs site, such as Developer Build Documentation.

**Comment by bogdan-calapod** [comment]
Opened #135. Currently improved the app listing and user guide
